diff --git a/node_modules/@snippetors/vuepress-plugin-code-copy/lib/clientAppSetup.js b/node_modules/@snippetors/vuepress-plugin-code-copy/lib/clientAppSetup.js
index 5a41e73..478e9fc 100644
--- a/node_modules/@snippetors/vuepress-plugin-code-copy/lib/clientAppSetup.js
+++ b/node_modules/@snippetors/vuepress-plugin-code-copy/lib/clientAppSetup.js
@@ -25,7 +25,7 @@ export default defineClientAppSetup(() => {
           };
           let instance = createApp(CodeCopy, {
             parent: el,
-            code: el.querySelector("pre").innerText,
+            code: el.querySelector("pre"),
             options: options,
           });
           let childEl = document.createElement("div");
diff --git a/node_modules/@snippetors/vuepress-plugin-code-copy/lib/components/CodeCopy.vue b/node_modules/@snippetors/vuepress-plugin-code-copy/lib/components/CodeCopy.vue
index 2506da8..e6087ab 100644
--- a/node_modules/@snippetors/vuepress-plugin-code-copy/lib/components/CodeCopy.vue
+++ b/node_modules/@snippetors/vuepress-plugin-code-copy/lib/components/CodeCopy.vue
@@ -1,5 +1,21 @@
 <template>
   <div class="code-copy">
+      <svg
+      @click="copyToClipboard"
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      custom="custom"
+      viewBox="0 0 24 24"
+      :class="iconClass"
+      :style="alignStyle"
+    >
+      <path fill="none" d="M0 0h24v24H0z" />
+      <path
+        :fill="options.color"
+        d="M 5.198 19.667 L 5.198 19.667 L 4.557 21.914 L 6.917 21.914 L 8.301 17.063 L 9.553 16.013 L 12.928 16.013 L 11.245 21.914 L 13.605 21.914 L 15.289 16.013 L 20.12 16.013 L 20.12 13.652 L 15.966 13.652 L 17.02 9.954 L 21.049 9.954 L 21.049 7.594 L 19.586 7.594 L 22.156 5.438 L 21.191 4.289 L 2.806 19.716 L 3.771 20.865 L 5.198 19.667 Z  M 12.366 13.652 L 13.605 13.652 L 13.995 12.286 L 12.366 13.652 Z  M 9.777 11.901 L 10.332 9.954 L 10.332 9.954 L 12.097 9.954 L 14.91 7.594 L 11.009 7.594 L 12.582 2.086 L 10.222 2.086 L 8.648 7.594 L 3.88 7.594 L 3.88 9.954 L 3.88 9.954 L 7.971 9.954 L 6.917 13.652 L 2.951 13.652 L 2.951 16.013 L 4.877 16.013 L 9.777 11.901 L 9.777 11.901 Z  M 15.471 7.123 L 16.91 2.086 L 19.27 2.086 L 18.575 4.518 L 15.471 7.123 L 15.471 7.123 Z"
+      />
+    </svg>
     <svg
       @click="copyToClipboard"
       xmlns="http://www.w3.org/2000/svg"
@@ -25,6 +41,7 @@
 </template>
 
 <script>
+import { cloneVNode } from '@vue/runtime-core'
 export default {
   name: "CodeCopy",
   props: {
@@ -78,8 +95,11 @@ export default {
         : null;
     },
     copyToClipboard(el) {
+      let stripped_code = this.code.cloneNode(true);
+      stripped_code.querySelectorAll(".hljs-comment").forEach(e => {e.remove()});
+      stripped_code = stripped_code.innerText.replace(/,\s*/gm, ", ").replace(/\(\n\s*/gm, "(").replace(/\s*\n\)/gm, ")").replace(/\n{3,}/gm, "\n\n");
       if (navigator.clipboard) {
-        navigator.clipboard.writeText(this.code).then(
+        navigator.clipboard.writeText(el?.srcElement?.attributes?.custom ? stripped_code : this.code.innerText).then(
           () => {
             this.setSuccessTransitions();
           },
@@ -88,7 +108,7 @@ export default {
       } else {
         let copyelement = document.createElement("textarea");
         document.body.appendChild(copyelement);
-        copyelement.value = this.code;
+        copyelement.value = el?.srcElement?.attributes?.custom ? stripped_code : this.code.innerText;
         copyelement.select();
         document.execCommand("Copy");
         copyelement.remove();
@@ -120,9 +140,20 @@ export default {
 </script>
 
 <style scoped>
+path {
+  pointer-events: none;
+}
+
+.code-copy > svg:first-child {
+  right: 7.5px;
+}
+
+.code-copy > svg {
+  right: 40px;
+}
+
 svg {
   position: absolute;
-  right: 7.5px;
   opacity: 0.75;
   cursor: pointer;
   z-index: 9999;
@@ -140,7 +171,7 @@ span {
   position: absolute;
   font-size: 0.85rem;
   line-height: 1.2rem;
-  right: 40px;
+  right: 80px;
   opacity: 0;
   transition: opacity 500ms;
 }
