diff --git a/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js b/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js
index 6f6a928..52b6395 100644
--- a/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js
+++ b/node_modules/@vuepress/plugin-active-header-links/lib/client/composables/useActiveHeaderLinks.js
@@ -12,7 +12,7 @@ export const useActiveHeaderLinks = ({ headerLinkSelector, headerAnchorSelector,
         // get all header anchors
         const headerAnchors = Array.from(document.querySelectorAll(headerAnchorSelector));
         // filter anchors that do not have corresponding links
-        const existedHeaderAnchors = headerAnchors.filter((anchor) => headerLinks.some((link) => link.hash === anchor.hash));
+        const existedHeaderAnchors = headerAnchors; //headerAnchors.filter((anchor) => headerLinks.some((link) => link.hash === anchor.hash));
         // get current scrollTop
         const scrollTop = Math.max(window.pageYOffset, document.documentElement.scrollTop, document.body.scrollTop);
         // get current scrollBottom
@@ -64,7 +64,11 @@ export const useActiveHeaderLinks = ({ headerLinkSelector, headerAnchorSelector,
     };
     const onScroll = debounce(setActiveRouteHash, delay);
     onMounted(() => {
-        onScroll();
+        let hash = window.location.hash;
+        onScroll().then(() => {
+            window.location.hash = "";
+            window.location.hash = hash;
+        })
         window.addEventListener('scroll', onScroll);
     });
     onBeforeUnmount(() => {
