diff --git a/node_modules/@vuepress/theme-default/lib/client/components/HomeHero.vue b/node_modules/@vuepress/theme-default/lib/client/components/HomeHero.vue
index 7f43bf0..a9f3dc5 100644
--- a/node_modules/@vuepress/theme-default/lib/client/components/HomeHero.vue
+++ b/node_modules/@vuepress/theme-default/lib/client/components/HomeHero.vue
@@ -60,6 +60,8 @@ const actions = computed(() => {
 const HomeHeroImage: FunctionalComponent = () => {
   if (!heroImage.value) return null
   const img = h('img', {
+    height: 280,
+    width: 280,
     src: withBase(heroImage.value),
     alt: heroAlt.value,
   })
